src/index.css

```diff
@@ -155,4 +155,38 @@ body {
   .notesContainer {
     width: 100%;
   }
+}
+
+
+.wrapper,
+.editor {
+  height: 70vh;
+  overflow: none;
+  flex: 1;
+  word-wrap: break-word;
+}
+
+.editor {
+  width: 100%;
+  flex: 1;
+  overflow-y: auto;
+  word-wrap: break-word;
+  /* border: 1px solid var(--border-color); */
```

Comment:
can remove this commented code from here.

Reply:
Yup on it.